src/main/webapp/teacher/executionCourse/evaluationMethod.jsp

```diff
@@ -58,20 +61,14 @@
 	</ul>
 	</logic:equal>
 
-	<h3 class="mtop15">
-		<bean:message key="title.evaluationMethod"/>
-	</h3>
-	
-	<blockquote>
-		<bean:write name="executionCourse" property="evaluationMethodText" filter="false"/>
-	</blockquote>
+    <c:forEach var="lang" items="<%= CoreConfiguration.supportedLocales() %>">
+        <h3 class="mtop15">
+            <bean:message key="${lang.language}" bundle="ENUMERATION_RESOURCES" />
+        </h3>
 
-	<h3 class="mtop2">
-		<bean:message key="title.evaluationMethod.eng"/>
-	</h3>
-	
-	<blockquote>
-		<bean:write name="executionCourse" property="evaluationMethodTextEn" filter="false"/>
-	</blockquote>
+        <blockquote>
+            ${evaluationMethods.getContent(lang)}
```

Comment:
Isn't it weird showing a whole section without content, if no content is defined for that locale? Should't we just iterate the locales in the actual string?